ignite/services/plugin/template/main.go.plush

```diff
@@ -8,20 +8,20 @@ import (
 
 
 	"github.com/ignite/cli/v29/ignite/services/plugin"
-	"<%= AppName %>/cmd"
+	"<%= ExtensionName %>/cmd"
 )
 
-type app struct{}
+type ext struct{}
```

Comment:
(Not a strong opinion) How about using "extn" instead of "ext" to avoid association with "external"?